tests/python/unittest/test_tir_schedule_analysis.py

```diff
@@ -41,6 +41,71 @@
 from tvm.te import create_prim_func
```

Comment:
Also remove change from this file

Reply:
There is no big changes, I tried to unify tests using the same classes, but my try failed and I return it back (as fact it was replaced inside the file). I've rollbacked the trasferred code. Just now there is pylint fix and renaming for the sake of clarity (not only VNNI is checked)